examples/GainPlugin/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ target_compile_definitions(GainPlugin
 # dependencies are resolved automatically, so `juce_core`, `juce_events` and so on will also be
 # linked automatically. If we'd generated a binary data target above, we would need to link to it
 # here too. This is a standard CMake command.
-target_link_libraries(GainPlugin PRIVATE
+target_link_libraries(GainPlugin PUBLIC
```

Comment:
I'm really a cmake newb still, what's the difference here?

Reply:
Ah I think that might be a left over error! Will fix.